orb/core/collection.py

```diff
@@ -625,8 +625,9 @@ def update(self, records, useMethod=True, **context):
                             record = self.__model.create(record_attributes, **context).id()
                         else:
                             record = self.__model(record_id, **context)
-                            record.update(record_attributes)
-                            record.save()
+                            if record_attributes:
+                                record.update(record_attributes)
+                                record.save()
```

Comment:
Q. Why update both sides of the pipe?  What do you mean?

Doing `record.update(record_attributes)` then `record.save()` should have not actually save anything if there are no changes.  If this is happening in our website, then it is a problem with the mixin creating changes when it shouldn't